src/components/AssignmentSummary.jsx

```diff
@@ -130,6 +130,15 @@ class AssignmentSummary extends Component {
               contactsFilter: 'reply',
               hideIfZero: true
             })}
+            {(window.NOT_IN_USA && window.ALLOW_SEND_ALL) ? '' : this.renderBadgedButton({
```

Comment:
I don't think we need to filter for NOT_IN_USA -- even non-US folks might want to revisit :-)